### DISCUSSION TITLE
I'm having trouble verifying my student status

### DISCUSSION THREAD

**Discussion by mariajorrat** [community/community#51808] [GitHub Education]
**I am frustrated with the terrible performance of the website. I have tried many times to verify my student account and the tool does not work. It does not recognize my university's URL, its name, mine, or even the enrollment certificate date.**

Have replied to me the following via email:

- _Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy._
**I have sent the documentation three times, I tried with a photo of my gradebook, with the enrollment certificate, and nothing works.**

- _You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so._
**This is outrageous! I literally live 10 blocks away from my university, I checked my IP location and it's correct.**

**My university does have a contract with GitHub, in fact, I consulted with them first and they told me that the problem is with GitHub's website.**

Best!

**Comment by samus-aran** [comment]
Hi @jorratmf 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Let's work through these rejection reasons: ⭐ 

1. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

2. We prefer live photos so if possible this would be the best way to submit your evidence.

3. If it is stating that you are far from campus can you identify that the campus is highlighted when you select your school. Does your campus have a different spelling, is it perhaps not been yet submitted or simply are you using a VPN.

Hope this helps! Let us know how you get along 👍

**Reply by mariajorrat** [reply]
Hello,

Thank you for your response and for trying to help me with the verification process. Unfortunately, I have tried the suggestions you provided and they did not work. I am still experiencing issues with the automatic verification process.

**As such, I would really appreciate it if someone from your team could assist me with the verification process manually. I need to be verified by a human as the automatic method is not functioning correctly.**

Thank you for your understanding and assistance.

Best regards.

**Reply by samus-aran** [reply]
Hi @jorratmf 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. **Do you have a screenshot of the rejection reasons?** This might help someone give you a nudge in the right direction.
